### DISCUSSION TITLE
After a month of use, colored PDF pages began to transform into black and white images with low quality

### DISCUSSION THREAD

**Discussion by borzovplus** [ImageMagick/ImageMagick#7753] [Help]
### ImageMagick version

8:6.9.11.60

### Operating system, version and so on

Ubuntu 22

### Description

Hello! After a month of use, colored PDF pages began to transform into black and white images with low quality. 
There were no changes or updates during this time, does it look like some kind of trial version?

 I use golang interface https://github.com/gographics/imagick
And Package: libmagickwand-dev
Version: 8:6.9.11.60+dfsg-1.3ubuntu0.22.04.5
Depends: imagemagick-6-common (= 8:6.9.11.60+dfsg-1.3ubuntu0.22.04.5), libmagickwand-6.q16-dev

Please help me.

**Comment by fmw42** [comment]
This is an Imagemagick forum, not an iMagick forum.  You might ask on an iMagick forum.

**Reply by fmw42** [reply]
Best quality is achieved using a high value of density (resolution) when reading such as 4x default (72) and then after reading resize down by the inverse amount such as 1/4=25%. Post an example PDF and your output.

**Comment by snibgo** [comment]
ImageMagick isn't iMagick.

ImageMagick doesn't have a "trial version".

ImageMagick uses Ghostscript to rasterize PDF documents. As far as I know, Ghostscript doesn't have a "trial version".

I suggest you post a PDF here, the ImageMagick command you used to rasterize it, and the result "black and white images with low quality".
